src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -6,19 +6,154 @@
  * За бажанням можна реалізувати інші методи інтрефейсу Map.</p>
  */
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    static final int DEFOULT_CAPASITY = 16;
+    static final double LOAD_FACTOR = 0.75;
+    private Entry<K, V>[] table;
+    private int size;
+
+    public <K, V> MyHashMap() {
+        table = new Entry[DEFOULT_CAPASITY];
+        size = 0;
+    }
 
     @Override
     public void put(K key, V value) {
```

Comment:
that's too complicated, try to simplify it

Reply:
Done